internal/ceb/config.go

```diff
@@ -112,12 +113,12 @@ func (ceb *CEB) watchConfig(
 	// env stores the currently known list of environment vars we set on the
 	// child. We need to store this since we want to launch all exec sessions
 	// with the latest/current view on env vars too.
```

Comment:
I think we need to update this to reflect the changes made. It now contains any updated configurations to the application (envvars & udpatedFiles). Same on line 174

Reply:
Yup, updated!